honeybee_schema/project_info.py

```diff
@@ -0,0 +1,91 @@
+"""Schema for project information."""
+from pydantic import BaseModel, Field, constr, AnyUrl
+from typing import List
+
+from .energy.simulation import EfficiencyStandards, ClimateZones, BuildingTypes
+
+
+class Location(BaseModel):
+    """A Ladybug Location."""
+
+    type: constr(regex='^Location$') = 'Location'
+
+    city: str = Field(
+        '-',
+        description='Name of the city as a string.'
+    )
+
+    latitude: float = Field(
+        0,
+        description='Location latitude between -90 and 90 (Default: 0).'
+    )
+
+    longitude: float = Field(
+        0,
+        description='Location longitude between -180 (west) and 180 (east) (Default: 0).'
+    )
+
+    time_zone: float = Field(
+        None,
```

Comment:
Not permissible by .NET. You have to use an AutoCalculate object for this to work on @MingboPeng 's end.

Reply:
AutoCalculate will work, otherwise the default time zone would be set to 0

Reply:
Thank you, @MingboPeng!